lldb/test/API/macosx/arm-pointer-metadata-stripping/main.c

```diff
@@ -0,0 +1,19 @@
+#include <assert.h>
+#include <stdint.h>
+#include <stdio.h>
+
+int myglobal = 41;
+
+uint64_t get_high_bits(void *ptr) {
+  uint64_t mask = ~((1ULL << 48) - 1);
```

Comment:
48 is the current default but we'd be a little safer if we just assume top byte, 56 bit VAs.

I know you have the PAC code on Arm64e but we know the pointer fix removes that so if it removes top byte in this test we know we're good.